pages/message-board.html

```diff
@@ -35,13 +34,50 @@
     <main>
       <h1>The Regis Altare Birthday Project 2023</h1>
 
+      <a href="#message-board" class="visually-hidden">Skip to content</a>
+
+      <!-- Inverted border radius for clip-path -->
+      <svg width="0" height="0" aria-hidden="true">
+          <clipPath id="navClip" clipPathUnits="objectBoundingBox">
+              <path transform="scale(0.004388274530454626, 0.019230769230769232)" d="M173.88,0H54C38,0,45,21,0,52H227.88C182.88,21,189.88,0,173.88,0Z"/>
+          </clipPath>
+      </svg>
+
+      <!-- Burger menu for responsive layout -->
+      <button id="nav-button" aria-label="Toggle site navigation">
+          <span class="visually-hidden">Menu</span>
+          <img class="icon icon-menu" aria-hidden="true" src="../icons/Icons_Hamburger.svg"/>
```

Comment:
(nit) If the button has aria-label, I think that will take precedence over the button contents, so I don't think the visually hidden text and aria-hidden is needed
I'm not on my Mac rn so can't confirm

Reply:
Did a quick test and can confirm that the aria-label takes precedence. Will remove the redundant span tag!